### PR TITLE
Add missing API tests for datarequest.

### PR DESCRIPTION
**Pull request by lwesterhof**

### CODE REVIEW

File: tests/features/api_datarequest.feature [hunk 1]
```diff
@@ -5,11 +5,78 @@ Feature: Datarequest API
...
+        And the result is "<result>"
+
+        Examples:
+            | user        | result |
+            | researcher  | False  |
+            | datamanager | False  |
+            | bodmember   | True   |
+            | dmcmember   | False  |
+
+    Scenario: Datarequest is dmc member
```
Comment (aucuparia) [inline tests/features/api_datarequest.feature]: dmc -> DMC

File: tests/features/api_datarequest.feature [hunk 2]
```diff
@@ -5,11 +5,78 @@ Feature: Datarequest API
...
+        And the result is "<result>"
+
+        Examples:
+            | user        | result |
+            | researcher  | False  |
+            | datamanager | True   |
+            | bodmember   | False  |
+            | dmcmember   | False  |
+
+    Scenario: Datarequest is bod member
```
Comment (aucuparia) [inline tests/features/api_datarequest.feature]: bod -> BoD

File: tests/features/api_datarequest.feature [hunk 3]
```diff
@@ -56,6 +123,20 @@ Feature: Datarequest API
         And the Yoda datarequest assignment get API is queried with request id
         Then the response status code is "200"
 
+    Scenario: Datarequest is reviewer
```
Comment (aucuparia) [inline tests/features/api_datarequest.feature:126]: A condition should be added that specifies that the user is assigned as a reviewer. E.g. `And "<user>" is assigned as reviewer of datarequest`.

Comment (lwesterhof) [reply tests/features/api_datarequest.feature:126]: This is handled in 'the result is "<result>"'?

**Review by aucuparia — APPROVED** [review]

**Pull request merged**